core/controllers/ingest.py

```diff
@@ -106,6 +109,25 @@ def extract_data(excel_file: FileStorage) -> dict[str, pd.DataFrame]:
     return workbook
 
 
+def normalize_data(workbook: dict[str, pd.DataFrame]):
```

Comment:
Can we make this a bit more of a pure function by returning the cleaned workbook rather than mutating it?